src/application/routes/__tests__/users.test.ts

```diff
@@ -323,4 +323,73 @@ describe('Routes: Users', () => {
       })
     })
   })
+
+  describe('when calling GET /users/getUser', () => {
```

Comment:
Ahh, aqui na verdade tem que mudar:

```suggestion
  describe('when calling GET /users/:id', () => {
```